apps/explorer-e2e/src/helpers/exploredStabilization.ts

```diff
@@ -0,0 +1,46 @@
+import { getExploredTip } from './getExploredTip'
+
+export async function exploredStabilization(
+  maxRetries = 20,
+  delayMs = 500,
+  stabilizationMs = 2000
+) {
+  for (let i = 0; i < maxRetries; i++) {
+    const tip = await getExploredTip()
+    const tipHeight = tip.data.height
+
+    console.log(
+      `observed explored tip height: ${tipHeight} — entering stabilization window`
+    )
+
+    const start = Date.now()
+    let isStable = true
+
+    while (Date.now() - start < stabilizationMs) {
+      const newTip = await getExploredTip()
+      const observedHeight = newTip.data.height
+
+      if (observedHeight !== tipHeight) {
+        console.log(
+          `explored height changed during stabilization — expected ${tipHeight}, got ${observedHeight}`
+        )
+        isStable = false
+        break
+      }
+
+      await new Promise((res) => setTimeout(res, 200))
+    }
+
+    if (isStable) {
+      console.log(`explored height stabilized at ${tipHeight}`)
```

Comment:
Where did we land on this, yesterday it looked like the height was tracking the cluster height?